references/product-recommendations-api/pipeline.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/sh
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+echo [INFO] Pipeline for product-recommendations-api in project: "$PROJECT_ID"
+
+# Set environment variables and default project for gcloud commands 
+source set_env_variables_my_apigeex.sh
+
+echo "[INFO] Pipeline for product-recommendations-api enable APIs"
+gcloud services enable bigquery.googleapis.com --project="$PROJECT_ID"
+gcloud services enable spanner.googleapis.com --project="$PROJECT_ID"
+
+echo "[INFO] Pipeline for product-recommendations-api - create service accounts"
+CURRENT_ACCOUNT=$(gcloud config get-value account)
+setup_service_accounts.sh
+
+echo "[INFO] Pipeline for product-recommendations-api - setup bigquery"
+setup_bigquery.sh
+
+echo "[INFO] Pipeline for product-recommendations-api - setup spanner"
+setup_spanner.sh
+
+echo "[INFO] Pipeline for product-recommendations-api - maven apigee install"
+# This performs end-to-end install, configuration and testing API.
+mvn -P eval clean install -Dbearer="$(gcloud auth print-access-token)" \
+    -DapigeeOrg="$ORG" \
+    -DapigeeEnv="$ENV" \
+    -DenvGroupHostname="$ENVGROUP_HOSTNAME" \
+    -DgcpProjectId="$PROJECT_ID" \
+    -DgoogleTokenEmail="$SA" \
+    -DcustomerUserId="$CUSTOMER_USERID"
+
+echo "[INFO] Pipeline for product-recommendations-api - get APIKEY"
+# Apigee API call to get API key for use in API calls.
+APIKEY=$(curl -s -H "Authorization: Bearer $(gcloud auth print-access-token)" \
+    https://apigee.googleapis.com/v1/organizations/"$ORG"/developers/demo@any.com/apps/product-recommendations-v1-app-"$ENV" \
+    | jq -r .credentials[0].consumerKey)
+
+echo "[INFO] Pipeline for product-recommendations-api - test api"
+# API call to show results
+curl -s "https://$ENVGROUP_HOSTNAME/v1/recommendations/products" \
```

Comment:
run integration tests instead of just a curl? Or is the apickli run as part of the mvn install? if yes then we don't need this curl at all.

Reply:
Apickli is run as part of mvn install, but doesn't show useful output.
Curl shows the full response.

Reply:
I'll remove